### DISCUSSION TITLE
Insert including all relations

### DISCUSSION THREAD

**Discussion by wertania** [directus/directus#17355] [Archive]
Hi there,
I´m new to directus cloud and I´m trying to evaluate if directus can replace some applications with smarter solutions as a backend.

Our own APIs had always the posibility to INSERT values including their relations.

That means if I have
a table A:
| companyName (unique) | companyId | address |
and a table B:
| employeeName | address | companyId

Then I would like to make POST on an employee. But not with the companyName and not the companyId.
And if the companyName is not exisiting then it should INSERT it automatically.
That´s saving a lot time in Frontend frameworks.

Is there a possibility to do that?

**Comment by rijkvanzanten** [comment]
> Is there a possibility to do that?

Yes! https://docs.directus.io/reference/introduction.html#relational-data
